### ISSUE TITLE
Can pedestrian images be data enhanced to improve their resolution?（是否可以对行人图像进行数据增强，提高其图像的分辨率？）

### ISSUE THREAD

**Issue by shams2023** [lose4578/SAM-DiffSR#1]
Can pedestrian images be data enhanced to improve their resolution?（是否可以对行人图像进行数据增强，提高其图像的分辨率？）

**Comment by lose4578** [comment]
Of course, this is a good direction to try. But currently, we focus on the real-wold scene and didn't have relevant experiments. If you have any relevant results, welcom to share them.

**Comment by shams2023** [comment]
> 当然，这是一个很好的尝试方向。但目前，我们专注于真实场景，没有相关的实验。如果您有任何相关结果，欢迎分享。

My pedestrian images are also obtained from real scenes (i.e. from cameras), but the resolution dial is generally low (blurry).
（我的行人图像也是从真实场景中获得的（即 摄像头获得的），只是分辨率盘普遍较低（模糊））
